File: scripts/testing/perf-workloads/crosslang/steady.js
function work(n) {
  const xs = []; for (let i = 1; i <= n; i++) xs.push(i);
  return xs.map(x => x + 1).length;
}
function repeat(times, n, acc) { return times <= 0 ? acc : repeat(times - 1, n, acc + work(n)); }
repeat(20, 50, 0);
const t0 = process.hrtime.bigint(); const r = repeat(200, 50, 0); const t1 = process.hrtime.bigint();
console.log(`node    elapsed_ms=${(Number(t1 - t0) / 1e6).toFixed(2)}  result=${r}`);

File: scripts/testing/perf-workloads/crosslang/steady.py
import time
def work(n):
    return len(list(map(lambda x: x + 1, range(1, n + 1))))
def repeat(times, n, acc):
    if times <= 0: return acc
    return repeat(times - 1, n, acc + work(n))
repeat(20, 50, 0)                      # warm
t0 = time.perf_counter(); r = repeat(200, 50, 0); t1 = time.perf_counter()
print(f"python  elapsed_ms={(t1-t0)*1000:.2f}  result={r}")
